.kitchen.yml

```diff
@@ -182,6 +182,38 @@ suites:
           backend: local
           controls:
             - shielded-vm
+  - name: private_connection
+    driver:
+      name: terraform
+      root_module_directory: test/fixtures/private_connection/
+      command_timeout: 1800
+    lifecycle:
+      pre_verify:
+        - chmod go-rwx test/fixtures/bastion/tls_private_key_private_conn
+    verifier:
+      systems:
+        - name: gcp
+          backend: gcp
+          controls:
+            - forseti
+        - name: gcloud
+          backend: local
+          controls:
+            - forseti-org-iam
+        - name: server
+          backend: ssh
+          controls:
+            - server
+          hosts_output: forseti-server-vm-ip
+          user: ubuntu
+          key_files:
+            - test/fixtures/private_connection/sshkey
+          proxy_command: ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -o IdentitiesOnly=yes -o LogLevel=ERROR -o ForwardAgent=n
```

Comment:
The build failed due to unable to parse this line. It looks like there is a line break in the `ForwardAgent=no`; try deleting the line break after the `n` in `no`.

Reply:
you are right. found it too.